src/server/visibility/destroy.ts

```diff
@@ -0,0 +1,28 @@
+import 'server-only'
+import { prismaCall } from '@/server/prismaCall'
+import prisma from '@/prisma'
+
+/**
+ * Destroy a visibility. This will also destroy the visibility levels associated with the visibility,
+ * to prevent orphaned visibility levels.
+ * @param id - The id of the visibility to destroy
+ */
+export async function destroyVisibility(id: number): Promise<void> {
+    const visibility = await prismaCall(() => prisma.visibility.delete({
```

Comment:
Should it be possible to delete a special visibility?

Reply:
That is actually extremely dangerous at it would cascade destroy everything connected to that purpose.